webservices/common/models/elections.py

```diff
@@ -10,6 +10,8 @@ class ElectionResult(db.Model):
     cand_office = db.Column(db.String, primary_key=True, doc=docs.OFFICE)
     cand_office_st = db.Column(db.String, primary_key=True, doc=docs.STATE_GENERIC)
     cand_office_district = db.Column(db.String, primary_key=True, doc=docs.DISTRICT)
+    election_type = db.Column(db.String)
+    fec_election_yr = db.Column(db.Integer)
```

Comment:
Just a note here:  we don't need any doc strings for these as they're not exposed in the API?  If we do we can file another PR with them later, I see no reason to hold this up.